cmd/influxd/main.go

```diff
@@ -36,6 +38,7 @@ const (
 
 func main() {
 	log.SetFlags(0)
+	rand.Seed(time.Now().UnixNano())
```

Comment:
We generate a new cluster id in `influxd restore` since we're not snapshotting the broker.